corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -266,6 +275,98 @@ public int dropTable(String namespace, String tablename) {
         return tableSize;
     }
 
+    public int editRecord(String namespace, String tableName,
+        String keyToEdit, String newRecord) {
+        verifyNamespaceAndTablename(namespace, tableName);
+
+        if (keyToEdit == null) {
+            throw new IllegalArgumentException("Please specify the key to " +
+                "edit --keyToEdit");
+        }
+
+        if (newRecord == null) {
+            throw new IllegalArgumentException("Please specify the newRecord " +
+                "--newRecord");
+        }
+
+        log.info("\n======================\n");
+        String fullName = TableRegistry.getFullyQualifiedTableName(namespace,
+            tableName);
+        UUID streamUUID = UUID.nameUUIDFromBytes(fullName.getBytes());
```

Comment:
If the format of StreamName -> UUID changes then this will be inconsistent. Use `CorfuRuntime::getStreamID` instead